packages/appstore/src/components/options/options.scss

```diff
@@ -0,0 +1,138 @@
+.options-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1.5rem;
+    width: 115.2rem;
+    height: 30.3rem;
+    @include mobile {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        position: absolute;
+        width: 32.8rem;
+        height: 28rem;
+    }
+
+    &__title-description-container {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        @include mobile {
+            width: 32.8rem;
+            height: 5.4rem;
+            font-style: normal;
+            font-weight: 400;
+            font-size: 1.2rem;
+            line-height: 1.8rem;
+        }
+        &--title {
+            line-height: 3.6rem;
+            color: $color-black-1;
+        }
+        &--description {
+            align-items: flex-start;
+            .link {
+                height: 2.4rem;
+                font-style: normal;
+                font-weight: 400;
+                font-size: 1.6rem;
+                line-height: 2.4rem;
+                text-decoration-line: underline;
+                color: $color-red;
+            }
+        }
+    }
+    &__accounts-platform-container {
+        display: flex;
+        flex-direction: column;
+        height: 21.9rem;
+        background: #ffffff;
+        border-radius: 1.6rem;
+        @include mobile {
+            display: flex;
+            flex-direction: column;
+            align-items: flex-start;
+            width: 32.8rem;
+            height: 5.6rem;
+            border-radius: 0rem 0.6rem 0rem 0rem;
+            margin-left: 1rem;
+        }
+        &--accounts {
+            display: flex;
+            flex-direction: row;
+            align-items: flex-start;
+            height: 7.2rem;
+            //max-width: 105rem;
+            //overflow-x: auto;
+            @include mobile {
+                display: flex;
+                flex-direction: row;
+                align-items: center;
+                padding: 0.8rem 1.6rem 0.8rem 0.8rem;
+                gap: 0.4rem;
+                isolation: isolate;
+                width: 13.5rem;
+                height: 6.2rem;
+                background: #f2f3f4;
```

Comment:
if this is available in `packages/shared/src/styles/constants.scss` reuse it, if not add it and reuse it